src/Microsoft.SqlTools.ServiceLayer/TableDesigner/TableDesignerService.cs

```diff
@@ -72,7 +72,7 @@ public void InitializeService(ServiceHost serviceHost)
 
         internal Task UpdateSettings(SqlToolsSettings newSettings, SqlToolsSettings oldSettings, EventContext eventContext)
         {
-            Settings.PreloadDatabaseModel = newSettings.MssqlTools.TableDesigner.PreloadDatabaseModel;
+            Settings.PreloadDatabaseModel = newSettings.MssqlTools.TableDesigner != null ? newSettings.MssqlTools.TableDesigner.PreloadDatabaseModel : false;
```

Comment:
Were you seeing an issue here?

FYI @alanrenmsft 
```suggestion
            Settings.PreloadDatabaseModel = newSettings.MssqlTools.TableDesigner?.PreloadDatabaseModel ?? false;
```

Reply:
when I was debugging getting the requests from mssql-vscode

Reply:
Ah yes, that'd do it. Definitely something to keep in mind when adding other settings so we don't end up accidently breaking VS Code MSSQL. Thanks for fixing!